gsi-13.xml

```diff
@@ -0,0 +1,61 @@
+  <remove-project name="platform/system/linkerconfig" />
+  <remove-project name="platform/hardware/interfaces" />
+  <remove-project name="platform/frameworks/libs/net" />
+
+
+  <project groups="gsi,capyloon"  name="platform_build" path="build/make" remote="capyloon-gsi" revision="gonk_android-13.0.0_r24-gsi">
+    <copyfile src="core/root.mk" dest="Makefile"/>
+    <linkfile src="CleanSpec.mk" dest="build/CleanSpec.mk"/>
+    <linkfile src="buildspec.mk.default" dest="build/buildspec.mk.default"/>
+    <linkfile src="core" dest="build/core"/>
+    <linkfile src="envsetup.sh" dest="build/envsetup.sh"/>
+    <linkfile src="target" dest="build/target"/>
+    <linkfile src="tools" dest="build/tools"/>
+  </project>
+  <project groups="gsi,capyloon" remote="capyloon-gsi" revision="gonk_android-13.0.0_r24-gsi" name="platform_system_core" path="system/core" />
+  <project groups="gsi,capyloon" remote="capyloon-gsi" revision="gonk_android-13.0.0_r24-gsi" name="platform_system_vold" path="system/vold" />
+
+  <!-- TODO: fork to capyloon-gsi -->
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_frameworks_native" path="frameworks/native" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_external_selinux" path="external/selinux" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_bootable_recovery" path="bootable/recovery" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_frameworks_av" path="frameworks/av" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_frameworks_base" path="frameworks/base" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_system_extras" path="system/extras" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_frameworks_opt_telephony" path="frameworks/opt/telephony" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_system_netd" path="system/netd" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_system_nfc" path="system/nfc" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_system_bpf" path="system/bpf" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_bionic" path="bionic" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_system_linkerconfig" path="system/linkerconfig" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_hardware_interfaces" path="hardware/interfaces" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_frameworks_libs_net" path="frameworks/libs/net" />
```

Comment:
There is another problem: these repos  added phh`s patches, which are compatible with more non-standard devices running GSI. Due to lack of time, I couldn't weed out patches that weren't necessary for B2G. So many repositories, I'm not sure there's a need to fork back to Capyloon.

Reply:
Yes, I checked out those b2g related forks to make sure that the GSI patches doesn't affect the work of b2g